core/src/scene/sceneLoader.cpp

```diff
@@ -990,6 +992,24 @@ void SceneLoader::loadSource(const std::shared_ptr<Platform>& platform, const st
         url = urlStream.str();
     }
 
+    auto constructUrls = [&](int loc, const std::string& subD) {
+        urls.push_back(url);
+        urls.back().replace(loc, 3, subD);
+    };
+
+    if (Node subDomainNode = source["url_subdomains"]) {
+        auto loc = url.find("{s}");
+        if (loc != std::string::npos && subDomainNode.IsSequence()) {
+            for (const auto& domain : subDomainNode) {
+                if (domain.IsScalar()) {
+                    constructUrls(loc, domain.Scalar());
```

Comment:
Ahh .. I just pushed a fix for the above (second comment)

With respect to subdomain substitution being similar to tile id substitution, I did think of doing it in the same way, but subdomains set is a **constant** **small** set, which does not need to be substituted in every request (which is obviously not true for tile ids).